signup-service/src/main/java/io/mosip/signup/helper/NotificationHelper.java

```diff
@@ -49,6 +49,9 @@ public class NotificationHelper {
     @Value("#{${mosip.signup.sms-notification-template.encoded-langcodes}}")
     private List<String> encodedLangCodes;
 
+    @Value("${mosip.signup.identifier.prefix}")
+    private String countryCode;
```

Comment:
this may not be country code in all the scenarios, so it will apt to just call this as "identifierPrefix"